spec/jobs/import_twilio_data_job_spec.rb

```diff
@@ -13,7 +13,9 @@
   end
 
   it 'performs the job - with correctly parsed date value' do
-    expect_any_instance_of(Importers::Twilio::Importer).to receive(:import).with(
+    rake_importer = double
```

Comment:
I'm confused why this spec (for a job) now refers to rake? Is that an oversight?